src/Post.php

```diff
@@ -118,7 +118,7 @@ public function topic()
      */
     public function user()
     {
-        return $this->belongsTo(User::class);
+        return $this->belongsTo(config('canvas.user', User::class));
```

Comment:
I believe we can remove the `User::class` here now, since we'll default the config variable to the User.